cdisc_rules_engine/services/define_xml_reader.py

```diff
@@ -278,9 +284,18 @@ def validate_schema(self) -> bool:
         """
         Validates Define XML Schema.
         """
-        logger.info(f"Validating Define-XML schema.")
+        logger.info("Validating Define-XML schema.")
         schema_validator = MetadataSchema()
         study = self._odm_loader.Study()
         is_valid: bool = schema_validator.check_conformance(study.to_dict(), "Study")
         logger.info(f"Validated Define-XML schema. is_valid={is_valid}")
         return is_valid
+
+    def get_define_version(self):
+        """Use to extract DefineVersion from file"""
+        self.read()
```

Comment:
Is it necessary to call .read() here?

Reply:
Also, add typing please

Reply:
calling read causes loading of the structure. without it we won't be able to access mdv attribute which contains information we need

Reply:
got it